samples/bluetooth/mesh/light_dimmer/prj.conf

```diff
@@ -49,7 +49,11 @@ CONFIG_BT_MESH_GATT_PROXY=y
 CONFIG_BT_MESH_PROXY_USE_DEVICE_NAME=y
 CONFIG_BT_MESH_DK_PROV=y
 CONFIG_BT_MESH_NLC_PERF_CONF=y
-CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE=1
+
+# Disable model extension API and set list size to 0 as the models in this sample don't have any
+# relation between each other.
+CONFIG_BT_MESH_MODEL_EXTENSIONS=n
+CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE=0
```

Comment:
This is not needed with changes done in the other PR.

Reply:
The list size is non-zero by default. The extension API is enabled by NLC profile Kconfig option:
https://github.com/nrfconnect/sdk-nrf/blob/e9f512477c9243727e2d391d05f7d4acee124008/subsys/bluetooth/mesh/Kconfig#L54-L55

Reply:
What I meant was that adding "https://github.com/nrfconnect/sdk-zephyr/pull/1536#pullrequestreview-1903761572" will the CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE be default = 0, and then not needed.

Reply:
The reason why the default value is not 0 is that then any attempt of adding a new entry to the list will fail. But perhaps this should be changed. Anyway, it should be done in upstream first.